libs/blocks/notification/notification.js

```diff
@@ -376,10 +380,23 @@ async function decorateLayout(el) {
   return foreground;
 }
 
+function setStickyAccessabilityAttributes(el) {
+  const section = el?.closest('.section');
+  if (!section) return;
+
+  setTimeout(() => {
+    const stickyClass = [...section.classList]
```

Comment:
NIT: No need for find
```suggestion
    const stickyClass = section.classList.contains('sticky-top') || section.classList.contains('sticky-bottom')
```